### ISSUE TITLE
Wrong shadow for style 6 coloured folder-music

### ISSUE THREAD

**Issue by dirtydancing** [numixproject/numix-folders#15]
Hi, I noticed that for Curvy Design No. 5, the symbol drop shadow colour for `folder-music` in the non-default folder colours in sizes 32, 64, 128 and 256 is orange instead of grey as everywhere else.

Example screenshots of grey-folder-music and of purple-folder-music in size 256:

![grey-folder-music](https://cloud.githubusercontent.com/assets/7164227/6433058/5ba3e140-c065-11e4-970b-c28d4f9986da.png)
![purple-folder-music](https://cloud.githubusercontent.com/assets/7164227/6433059/5ec506e2-c065-11e4-9811-842462e95a18.png)

In contrast, the colour of the symbol drop shadow for the default folder colour is fine:

![default-folder-music](https://cloud.githubusercontent.com/assets/7164227/6433077/b817fd30-c065-11e4-9597-21896600809e.png)

**Comment by Foggalong** [comment]
@dirtydancing You're probably the most prolific issue reporter we've had for Numix. We've been given another week until 'Folder Color' is ready to release this collaboration, so in the meantime could we ask you to test styles 1-4, & 6 as you have for 5 we would be incredibly greatful <3

**Comment by dirtydancing** [comment]
Thank you very much, I am glad to experience how Numix is integrating input from the community!

And sure, I am happy to look into this :-)

**Comment by dirtydancing** [comment]
@Foggalong Ok, I believe by now with https://github.com/numixproject/numix-folders/issues/27 I have finished my testing of folder styles 1-6.

I have to say: Wow, this is beyond awesome! And I have been asking myself: how did you manage to not go insane or "folder-blind" with all these thousands and thousands of folder icons? :-)

Actually I did like all 6 folder styles, but for my personal taste, still nothing beats style No. 5 in default colour, and with nothing I mean any folder design I have ever seen so far. Supremely done!
